src/playlist.js

```diff
@@ -105,11 +105,12 @@
       if (!playlist.endList) {
         return window.Infinity;
       }
+      endSequence = playlist.segments.length;
```

Comment:
In this case, it is being used as a "sequence" for the purposes of the calculation later. It seems silly to introduce another variable and/or a condition when a simple assignment will suffice.